Patches/DefaultAgeModelPatch.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using HarmonyLib;
+using TaleWorlds.CampaignSystem.SandBox.GameComponents;
+
+namespace BannerlordTweaks.Patches
+{
+    [HarmonyPatch(typeof(DefaultAgeModel), "BecomeInfantAge", MethodType.Getter)]
```

Comment:
Since you are only patching the model's getters.
Create a TweakedModel that inherits from the DefaultAgeModel instead then inject that.
Have a look at the TweakedPregnancyModel below.

Reply:
Afterwards you can also move it out of the patches folder since we are just using our own User configurable Model.

Reply:
yeah, I was not sure because of the Harmony Compatibility feature, I was just not sure how it will react to other mods doing the similar feauter change

Reply:
Good point, Harmony would end up patching the method/getter from the base class.
Which would only be called if our tweaks aren't active.
`        public override float PregnancyDurationInDays => Settings.Instance.PregnancyDurationTweakEnabled
            ? Settings.Instance.PregnancyDuration
            : base.PregnancyDurationInDays;
`

I still think we should proceed with the tweaked model, since there really isn't much use in having multiple patches override a single value getter.

In the case where we don't provide an overriden getter and there is a patch on that getter.
Then the base getter (patched) would be called.

Reply:
Since we give the user control whether he wants to have our tweaked model or not.
It makes sense that if he enabled it, that we have priority over other mods.
Which we guarantee by using a Tweaked Model instead of a harmony patch.

Reply:
well then it is clear to me, making the changes.